src/v2/Apps/Home/__tests__/HomeApp.jest.tsx

```diff
@@ -0,0 +1,69 @@
+import React from "react"
+import { graphql } from "relay-runtime"
+import { MockBoot } from "v2/DevTools"
+import { setupTestWrapper } from "v2/DevTools/setupTestWrapper"
+import { HomeAppFragmentContainer } from "../HomeApp"
+import { HomeApp_Test_Query } from "v2/__generated__/HomeApp_Test_Query.graphql"
+import { useSystemContext } from "v2/System/useSystemContext"
+
+jest.mock("v2/System/useSystemContext")
+jest.unmock("react-relay")
+
+describe("HomeApp", () => {
+  const { getWrapper } = setupTestWrapper<HomeApp_Test_Query>({
+    Component: ({ homePage }) => (
+      <MockBoot>
+        <HomeAppFragmentContainer homePage={homePage!} />
+      </MockBoot>
+    ),
+    query: graphql`
+      query HomeApp_Test_Query {
+        homePage {
+          ...HomeApp_homePage
+        }
+      }
+    `,
+  })
+
+  describe("logged out", () => {
+    beforeAll(() => {
+      ;(useSystemContext as jest.Mock).mockImplementation(() => ({
```

Comment:
Maybe 
```tsx 
const mockuseSystemContext = useSystemContext as jest.Mock
```

as we'll likely want to add future tests and 
```tsx
;(useSystemContext as jest.Mock)
``` 

could get a bit old